docs/contributors/release.md

```diff
@@ -209,7 +209,7 @@ The Gutenberg repository mirrors the [WordPress SVN repository](https://make.wor
 
 ### Synchronizing WordPress Trunk
 
-For each Gutenberg plugin release, WordPress trunk should be synchronized with this release. This involves the following steps that are automated with `./bin/commander npm-stable` command:
+For each Gutenberg plugin release, the WordPress trunk should be synchronized with this release. This involves the following steps that are automated with `./bin/commander npm-stable` command:
```

Comment:
This refers to the `trunk` folder in SVN, _not_ "the trunk" as in "the trunk of a car", so this has different meaning.

Let's leave this as "WordPress trunk"